src/edu/csus/ecs/pc2/clics/API202306/AccountService.java

```diff
@@ -0,0 +1,149 @@
+        this.model = inContest;
+        this.controller = inController;
+    }
+
+    /**
+     * This method returns a representation of the current model teams in JSON format. The returned value is a JSON array with one team description per array element.
+     * 
+     * @param sc user information
+     * @param contestId the contest for which the teams are requested
+     * @return a {@link Response} object containing the model teams in JSON form
+     */
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getAccounts(@Context SecurityContext sc, @PathParam("contestId") String contestId, @PathParam("svc") String svc) {
+
+        // check contest id
+        if(contestId.equals(model.getContestIdentifier()) == false) {
+            return Response.status(Response.Status.NOT_FOUND).build();        
+        }
+        
+        String myUser = sc.getUserPrincipal().getName();
+        
+        if(svc.equals("accounts")) {
+            ArrayList<CLICSAccount> alist = new ArrayList<CLICSAccount>();
+            
+            for(Account account: model.getAccounts()) {   
+                
+                // Admin can see everyone, everyone else only sees themselves.
+                if (sc.isUserInRole(WebServer.WEBAPI_ROLE_ADMIN) || myUser.equals(account.getClientId().getName())) {
+                    alist.add(new CLICSAccount(model, sc, account));
+                }
+            }
+            try {
+                ObjectMapper mapper = JSONUtilities.getObjectMapper();
+                String json = mapper.writeValueAsString(alist);
+                return Response.ok(json, MediaType.APPLICATION_JSON).build();
+            } catch (Exception e) {
+                return Response.status(Status.INTERNAL_SERVER_ERROR).entity("Error creating JSON for accounts " + e.getMessage()).build();
+            }
+        } else {
+            // account endpoint - who am i, basically.
+            for(Account account: model.getAccounts()) {
+                // Looking for currently authenticated user
+                if(myUser.equals("" + account.getClientId().getName())){
+                    return Response.ok(new CLICSAccount(model, sc, account).toJSON(), MediaType.APPLICATION_JSON).build();
+                }
+            }
+            return Response.status(Response.Status.NOT_FOUND).build();
+        }
+    }
+
+    /**
+     * Return response to the request for information about a specific teamid in a specific contestid
+     * 
+     * @param sc user information
+     * @param contestId the contest
+     * @param accountId the account id
+     * @return response
+     */
+    @GET
+    @Produces({ MediaType.APPLICATION_JSON })
+    @Path("{accountId}/")
+    public Response getTeam(@Context SecurityContext sc, @PathParam("contestId") String contestId, @PathParam("accountId") String accountId) {
+
+        // check contest id
+        if(contestId.equals(model.getContestIdentifier()) == true) {
+            
```

Comment:
The `== true` test seems redundant ?   (`.equals()` returns a boolean, yes?   It's not like JavaScript where things can be `truthy` or `falsey`...)

Reply:
We discussed this, and while they could be removed, I think we decided if we're editing a module that has it and we notice it, we'll remove it, otherwise, I'll be aware of this when writing future code.  Old habits die hard.